src/views/HomePage.vue

```diff
@@ -2,6 +2,32 @@
 <script setup>
 import { RouterLink } from "vue-router";
 import NavBar from "@/components/NavBar.vue";
+import { store } from "../store.js"
+import AnimalCardsSection from '@/components/AnimalCardsSection.vue'
+</script>
+
+<script>
+export default {
+  name: "HomePage",
+  data() {
+    return {
+      store
+    }
+  },
+  created() {
```

Comment:
Just like componentDIdMount

Reply:
I feel like this was a little too much on this branch, but I understand why we did it.